### ISSUE TITLE
Package Deploy continuously checking expired SAS URL and all functions will gone.

### ISSUE THREAD

**Issue by btardif** [Azure/azure-functions-host#3404]
_From @guitarrapc on September 4, 2018 17:49_

### Investigative information

Please provide the following:

Timestamp: 2018-09-04T02:24:00.509Z
Function App version (1.0 or 2.0-beta): 2.0-beta (2.0.12050-alpha)
Function name(s) (as appropriate):
Invocation ID: Operation ID gjlG5NmWiiI: not a invokation issue.
Region: Japan East

### OS and Browser

Not OS, Browser issue.

### Repro steps

1. Deploy your awesome function zip to public accessible BLOB SAS url.
1. Set your function zip url to app settings `WEBSITE_RUN_FROM_ZIP=<MY FUNCTION ZIP URL>`
1. Confirm functions are deployed.
1. expire SAS url.
1. Deployed Functions will be removed.

### Actual behavior

My functions are using Package Deployment with referencing BLOB SAS url. It successfully deployed/work as expected, but all functions disappeared when SAS expired and URL in inaccessible.

```
WEBSITE_RUN_FROM_ZIP=<MY FUNCTION ZIP URL>
```

### Expected behavior

Once valid functions are pulled and deploy via Package Deployment, although url is expired later but functions should keep as they are deployed. 

* In other words, should stop monitoring until url is changed
* In case this is expected behavior for some reason, add some userside controllable method to  to keep functions. Such as adding extra parameter?
    * Ex. WEBSITE_KEEP_FUNCTIONS_RUN_FROM_ZIP = 1

### Any error messages

Not a error, but ux proposal.

### Mention any other details that might be useful.

If I use ZipDeployment + Package Deployment, it never happen, off course 😄 

_Copied from original issue: Azure/azure-functions-ux#3071_

**Comment by btardif** [comment]
@davidebbo I don't think this is UX issue. This is a run-from-zip issue.

Please take a look

**Comment by davidebbo** [comment]
Yes, this is unrelated to Portal, and unrelated to Functions.

It is by design that when running from a package using an external URL, the URL needs to remain valid. It is not a deployment time URL, but it is used at runtime each time the app starts.

**Comment by guitarrapc** [comment]
Thank you, it makes sense.
